packages/kyt-core/src/config/webpack.dev.server.js

```diff
@@ -2,25 +2,51 @@
 
 const webpack = require('webpack');
 const nodeExternals = require('webpack-node-externals');
-const { serverSrcPath, serverBuildPath } = require('kyt-utils/paths')();
+const StartServerPlugin = require('start-server-webpack-plugin');
+const {
+  serverSrcPath,
+  serverBuildPath,
+  clientAssetsFile,
+  loadableAssetsFile,
+} = require('kyt-utils/paths')();
 const getPolyfill = require('./getPolyfill');
 
+const nodeArgs = ['-r', 'source-map-support/register'];
+// Passthrough --inspect and --inspect-brk flags (with optional [host:port] value) to node
+if (process.env.INSPECT_BRK) {
+  nodeArgs.push(process.env.INSPECT_BRK);
+} else if (process.env.INSPECT) {
```

Comment:
Should we warn if someone supplies both flags?

Reply:
copying this: https://github.com/jaredpalmer/razzle/blob/ef3041789b2c9c56c602991e9383e59e483cc666/packages/razzle/config/createConfig.js#L325

let's not worry about it for now